### PR TITLE
Traceability interop

### PR DESCRIPTION
**Pull request by mkhraisha**
Adds Traceability Interop spec following procedure in the readme

### CODE REVIEW

**Review by dontcallmedom — CHANGES_REQUESTED** [review]
* the fileame should be index.html rather than `unofficial-…`
* the `CG-FINAL-traceability…` directory should be part of the `credentials` directory (since this is report from the credentials CG)
* the "latest published version" URI is wrong
* conversely, you may want to set a "This version" URI

**Comment by mkhraisha** [comment]
@dontcallmedom
Updated filename,
Moved to credentials,
Updated URL for latest version I used https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204 as i believe that is where it will end up but please let me know if im supposed to do anything else

File: credentials/CG-FINAL-traceability-interop-20241204/index.html [hunk 1]
```diff
@@ -0,0 +1,1341 @@
...
+    }
+  
+</style>
+
+<meta name="color-scheme" content="light">
+<meta name="description" content="This specification describes discovery and exchange mechanisms of W3C Verifiable Credentials Data Model v2.0 [VC-DATA-MODEL],
+    particularly as they relate to supply chain use cases.
+    Exchanged Verifiable Credentials often implement schemas defined in
+    the Traceability Vocabulary [TRACE-VOCAB]..">
+<link rel="canonical" href="https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204">
```
Comment (dontcallmedom) [inline credentials/CG-FINAL-traceability-interop-20241204/index.html:116]:
```suggestion
<link rel="canonical" href="https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204">
```

File: credentials/CG-FINAL-traceability-interop-20241204/index.html [hunk 2]
```diff
@@ -0,0 +1,1341 @@
...
+    <p id="w3c-state">
+      <a href="https://www.w3.org/standards/types#reports">Final Community Group Report</a>
+      <time class="dt-published" datetime="2024-12-04">04 December 2024</time>
+    </p>
+    <dl>
+      <dt>This version:</dt><dd>
+              <a class="u-url" href="https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204/">https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204/</a>
+            </dd>
+      <dt>Latest published version:</dt><dd>
+              <a href="https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204">https://www.w3.org/community/reports/ccg/CG-FINAL-traceability-interop-20241204</a>
```
Comment (dontcallmedom) [inline credentials/CG-FINAL-traceability-interop-20241204/index.html:304]:
```suggestion
              <a href="https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204">https://www.w3.org/community/reports/credentials/CG-FINAL-traceability-interop-20241204</a>
```

**Review by dontcallmedom — APPROVED** [review]

**Pull request merged**